packages/aws-signing-helper/aws-signing-helper.spec

```diff
@@ -0,0 +1,49 @@
+%global goproject github.com/aws
+%global gorepo rolesanywhere-credential-helper
+%global goimport %{goproject}/%{gorepo}
+
+%global gover 1.0.2
+%global rpmver %{gover}
+
+%global _dwz_low_mem_die_limit 0
+
+%global gitrev b193b4e041c18896203b77724fa0577cf73949fa
```

Comment:
This is what needs to be replaced by a version tag, right?

Reply:
Here and in `Cargo.toml` will need to be updated to the official release tag. Then based on that, the source tar.gz needs to be added to our lookaside cache. Then we should be good to go.